azurepipelines/e2e_test/scenarios/ubuntu-18.04-arm64/vm_setup/setup.sh

```diff
@@ -164,6 +164,9 @@ function test_shutdown_service() {
 }
 
 configure_apt_repository
+
```

Comment:
why was this moved up?

Reply:
We need to have the APT repo configured before we do anything else so we know the APT repo on the machine has been configured with packages.microsoft.com

Reply:
We need to have the APT repo configured before we do anything else so we know the APT repo on the machine has been configured with packages.microsoft.com